packages/config/config/devices/0x010f/fgrm222_0.0-22.22.json

```diff
@@ -39,9 +39,6 @@
 			"isLifeline": true
 		}
 	},
-	"proprietary": {
-		"fibaroCCs": [38 /* 0x26 (Venetian Blinds) */]
-	},
```

Comment:
That must stay!

Reply:
Ok, no idea what it does and its not in the json chema so lint complains about it in VS Code...